core/src/main/java/org/infinispan/topology/LocalTopologyManagerImpl.java

```diff
@@ -106,15 +106,18 @@ public void start() {
       latestStatusResponseViewId = transport.getViewId();
    }
 
-   // Need to stop before the JGroupsTransport
-   @Stop(priority = 9)
+   // Need to stop after ClusterTopologyManagerImpl and before the JGroupsTransport
+   @Stop(priority = 110)
```

Comment:
We should really have proper dependency graphs to avoid these "conjured" numbers